test/TestProjects/SpecificVersion-TypeSpec/src/Generated/SpecificClientOptions.cs

```diff
@@ -18,10 +18,8 @@ public partial class SpecificClientOptions : ClientOptions
         /// <summary> The version of the service to use. </summary>
         public enum ServiceVersion
         {
-            /// <summary> Service version "2022-06-01-preview". </summary>
-            V2022_06_01_Preview = 1,
```

Comment:
This is expected. target api-version is `V2022-09-01` which is a stable api-version, so filter out all preview api-version.